securedrop/source_templates/index.html

```diff
@@ -62,7 +62,7 @@
     Included here so the images can preload while the user is first
     reading the page. Hidden by default. -->
     <div class="bubble">
-      <p>You appear to be using the Tor Browser. You can turn the Security Slider to High in 4 easy steps!</p>
+      <p>You appear to be using the Tor Browser. In Tor's "Security Settings" pane (accessible from the toolbar's onion-button), we reccommend turning Security Slider to High—it's easy-peasy!</p>
```

Comment:
`'onion-button), we reccommend '` -> `'onion button), we recommend'`
`'easy-peasy!'` -> `'easy!'`